api/client/gro_client.py

```diff
@@ -90,11 +91,14 @@ def main():
     parser.add_argument("--token")
     args = parser.parse_args()
 
-    assert args.user_email or args.token, \
+    assert args.user_email or args.token or os.environ['GROAPI_TOKEN'], \
         "Need --token, or --user_email"
     access_token = None
+
     if args.token:
         access_token = args.token
+    elif os.environ['GROAPI_TOKEN']:
```

Comment:
this will throw a key error when not defined.

instead, just use a argparse default value

Reply:
done

Reply:
retested  with the new changes
```
$ python gro_client.py --item potato --metric "production quantity mass"
Picking first result out of 291 items: 4
Picking first result out of 500 metrics: 860032
Data series example:
Using data series: {u'end_date': u'2017-06-30T00:00:00.000Z', u'source_name': u'IDAC', u'region_id': 117389, u'region_name': u'Howrah', u'item_name': u'Potatoes', u'partner_region_name': u'World', u'frequency_id': 9, u'source_id': 50, u'partner_region_id': 0, u'item_id': 4, u'metric_name': u'Production Quantity (mass)', u'start_date': u'1997-07-01T00:00:00.000Z', u'metric_id': 860032}
Outputing to file: gro_client_output.csv

```
looks good